packages/batch-poster-monitor/index.ts

```diff
@@ -623,6 +678,52 @@ const main = async () => {
   }
 }
 
+const checkIfAnyTrustRevertedToOnChain = async ({
+  parentChainClient,
+  childChainInformation,
+  lastSequencerInboxLog,
+}: {
+  parentChainClient: PublicClient
+  childChainInformation: ChainInfo
+  lastSequencerInboxLog:
+    | Log<bigint, number, false, AbiEvent, undefined, [AbiEvent], string>
+    | undefined
+}): Promise<string[]> => {
+  const alerts = []
+
+  // Get the transaction that emitted `lastSequencerInboxLog`
+  const transaction = await parentChainClient.getTransaction({
+    hash: lastSequencerInboxLog?.transactionHash as `0x${string}`,
+  })
+
+  const { args } = decodeFunctionData({
+    abi: sequencerInboxAbi,
+    data: transaction.input,
+  })
+
+  // Extract the 'data' field
+  const batchData = args[1] as `0x${string}`
+
+  // Check the first byte of the data
+  const firstByte = batchData.slice(0, 4)
+
+  if (firstByte === '0x00') {
+    alerts.push(
+      `AnyTrust chain [${childChainInformation.name}] has fallen back to posting calldata on-chain. This indicates a potential issue with the Data Availability Committee.`
+    )
+  } else if (firstByte === '0x88') {
+    console.log(
+      `Chain [${childChainInformation.name}] is using AnyTrust DACert as expected.`
```

Comment:
Would it be very noisy to log this for all chains? Not sure 🤔 . We can try and see how it goes.

Reply:
I just tested and I think we can leave it for now :)